main.js

```diff
@@ -10,46 +11,43 @@ const clear = () => {
   author.value = '';
 };
 
```

Comment:
On line 7 you have your books array outside of the class

![image](https://user-images.githubusercontent.com/38088818/191611194-52a3b012-541a-4e90-8d8f-043b0aa7a1b8.png)

- Kindly make sure that the use of objects and arrays is **inside** the classes.

Your class structure should be similar to this:

![image](https://user-images.githubusercontent.com/38088818/191610953-36dd1d41-a314-43c4-bec3-6883c24f0379.png)